### PR TITLE
Improve project

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Improve project (Renamed Readme.md / Added dependabot / Added template for PR / Added PHP Workflow & Build Release) & Removed Travis CI
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| How to test?      | Github Actions are :green_circle: https://github.com/Progi1984/gamification/actions

### CODE REVIEW

**Comment by NeOMakinG** [comment]
I think we should add new lines at the end of every files, don't you think?

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @Progi1984

File: controllers/admin/AdminGamificationController.php [hunk 1]
```diff
@@ -237,20 +236,20 @@ public function processAdviceValidation()
...
         $return = true;
-        $current_level = (int)Configuration::get('GF_CURRENT_LEVEL');
-        $current_level_percent = (int)Configuration::get('GF_CURRENT_LEVEL_PERCENT');
+        $current_level = (int) Configuration::get('GF_CURRENT_LEVEL');
+        $current_level_percent = (int) Configuration::get('GF_CURRENT_LEVEL_PERCENT');
 
-        $not_viewed_badge = explode('|', ltrim(Configuration::get('GF_NOT_VIEWED_BADGE', ''), ''));
-        $nbr_notif = Configuration::get('GF_NOTIFICATION', 0);
+        $not_viewed_badge = explode('|', ltrim(Configuration::get('GF_NOT_VIEWED_BADGE', null, null, null, ''), ''));
+        $nbr_notif = Configuration::get('GF_NOTIFICATION', null, null, null, 0);
```
Comment (eternoendless) [inline controllers/admin/AdminGamificationController.php:243]: Parameters are invalid here

File: gamification.php [hunk 2]
```diff
@@ -311,33 +309,49 @@ public function refreshDatas($iso_lang = null)
...
+                $public_key = file_get_contents(__DIR__ . '/prestashop.pub');
+                $signature = isset($data->signature) ? base64_decode($data->signature) : '';
+
+                if (isset($data->conditions)) {
+                    if (
+                        function_exists('openssl_verify')
+                        && self::TEST_MODE === false
+                        && isset($data->advices_lang)
+                        && !openssl_verify(json_encode([$data->conditions, $data->advices_lang]), $signature, $public_key)
+                        ) {
```
Comment (eternoendless) [inline gamification.php:321]: These changes haven't been accounted for?

File: controllers/admin/AdminGamificationController.php (hunk 1 above)
Comment (eternoendless) [reply controllers/admin/AdminGamificationController.php:243]: Ok my mistake

**Pull request merged**